instagram/src/App.js

```diff
@@ -10,16 +10,43 @@ class App extends React.Component {
     this.state = {
       // postData: dummyData,
       postData: [],
+      originalPostData: [],
+      searchField: ""
     };
   }
 
+  addNewComment = (event,index) => { 
+    event.preventDefault();
+    const comments = this.state.comments.slice();
+    const newComment = {
+        username: "default usrname",
+        text: this.state.comment
+    }
+    comments.push(newComment);
+    this.setState({comments: comments, comment: ""});
```

Comment:
nice work here.  with es6 shorthand you can change comments: comments into just comments.  But this works as well.  Good Job!